cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/1_create_datasource.spec.js

```diff
@@ -17,6 +17,27 @@ const REGION = 'us-east-1';
 const ACCESS_KEY = 'accessKey';
 const SECRET_KEY = 'secretKey';
 
+export const CreateDataSourceNoAuth = () => {
+  cy.request({
+    method: 'POST',
+    url: `${Cypress.config('baseUrl')}/api/saved_objects/data-source`,
+    headers: {
+      'osd-xsrf': true,
+    },
+    body: {
+      attributes: {
+        title: `9201`,
```

Comment:
What I had in mind for this function is adding a remote opensearch running on a different port to provide a helper function for plugins that want to run their own functional tests against an external opensearch cluster - hence hard coding this to http://localhost:9201, which is what I have put in the github action to spin up prior to running this - does that make sense? Although not used today I was hoping plugins that need it can use this wdyt @kavilla